dagger-core/src/main/java/io/odpf/dagger/core/processors/external/http/request/HttpPutRequestHandler.java

```diff
@@ -39,7 +42,7 @@ public BoundRequestBuilder create() {
         String endpoint = httpSourceConfig.getEndpoint();
 
         if (!StringUtil.isNullOrEmpty(httpSourceConfig.getEndpointVariables())) {
```

Comment:
You need not do this validation check anymore as endpointVariablesValues cannot be null but can be empty.
You can just do
`String endpoint = String.format(httpSourceConfig.getEndpoint(), endpointVariablesValues)`

at line 42.

Reply:
https://github.com/odpf/dagger/pull/217/files#diff-068c730b8c7790e141bb473ed08ae9ef6d60558d51288be41c10a5dcc9754321R67 I added one here @Meghajit